backpack/extensions/saved_quantities.py

```diff
@@ -0,0 +1,50 @@
+"""Class for saving backpropagation quantities."""
+from typing import Dict, Union
+
+from torch import Tensor
+
+
+class SavedQuantities:
+    """Implements interface to save backpropagation quantities."""
+
+    def __init__(self):
+        """Initialization."""
+        self._saved_quantities: Dict[int, Tensor] = {}
+
+    def save_quantity(self, key: int, quantity: Tensor) -> None:
+        """Saves the quantity under the specified key.
+
+        Args:
+            key: data_ptr() of reference tensor.
+                For torch>=1.9.0 the reference tensor is grad_input[0].
+                For older versions the reference tensor is module.input0.
```

Comment:
I think this is still incorrect (see [this line](https://github.com/tjs111/backpack/blob/fa09668cdc043982b89ac395ca77ef001a281a57/backpack/extensions/module_extension.py#L134). We always use `module.input0.data_ptr()` as identifier.

Reply:
Pushing a correction. Let me know if I missed something.